src/window/win.cc

```diff
@@ -649,6 +649,8 @@ namespace SSC {
     SetWindowLongPtr(window, GWLP_USERDATA, (LONG_PTR) this);
 
     SSC::String preload = createPreload(opts);
+    // store preload JS on window because, preload variable above goes out of scope by the time browser handlers are called.
+    this->preloadJavascript = createPreload(opts);
```

Comment:
Seems like a waste to call `createPreload(opts)` twice - just do `this->preloadJavascript = preload`